### ISSUE TITLE
The display of the 'Samples' card is not consistent with the display of the cards of other entities outside the user's jurisdiction

### ISSUE THREAD

**Issue by adinaflorea9** [hzi-braunschweig/SORMAS-Project#11519]
### Problem Description

When the person and its associated entities are outside the user's jurisdiction, the samples associated with them are no longer displayed on the 'Case' page ('Contact' page, 'Event participant' page), but on the 'Person' page the sample is displayed.

#### Steps to Reproduce

1. With an Admin+NatUser create a case with a sample associated in jurisdiction 1;
2. Copy the URL links of the person and the case created at step 1;
3. Log in with a configured user restricted by jurisdiction (jurisdiction 2);
4. Observe the 'Samples' card on the person and the case pages from the saved URLs.

### Actual Behavior

Sample on the 'Case' page when the entities are outside the user's jurisdiction:
![image](https://user-images.githubusercontent.com/96909231/219967086-c660265d-11aa-4601-9df6-af1ffa44803d.png)

Samples on the 'Person' page when the entities are outside the user's jurisdiction:
![image](https://user-images.githubusercontent.com/96909231/219967068-1e6738bb-220a-473b-8016-bf5b1664e8ec.png)

### Expected Behavior

@MateStrysewske is there perhaps a reason why the 'Samples' card is not displayed on the 'Case' page when the case is outside the user's jurisdiction?

Expected behavior proposed, depending on whether it is expected or not that the sample to not be displayed on the 'Case' page when the entity is outside the user's jurisdiction:

1. The sample should be displayed on the case page.

2. The sample should not be displayed on the person page.

#### System Details

* Device: Windows 10
* SORMAS version: 1.80.0; 1.81.0
* Android version/Browser: Chrome
* Server URL: local machine
* User Role: Admin'NatUser; configured user restricted by jurisdiction, but with all the rights of an Admin+NatUser

**Comment by bernardsilenou** [comment]
@adinaflorea9 
- According to your scenario, the user in district2 should not even be able to open the URL (assume you used another browser ) because they neither have full access nor pseudonymised access to the case. 
- `A user has access to a person if they have full access to at least one of the entities of the person`. The user should not also see this person + sample card
- If a user does not have access to parent (case ), then they do not also have access to child (sample)

**Comment by MateStrysewske** [comment]
@adinaflorea9 I guess the first and foremost issue here is that the person page is editable in the first place; if I understood your description correctly, the restricted user you're using does not have any entity related to that person in their jurisdiction, correct? That would mean that we're not properly managing the read/edit status of the person page when accessing it through the person directory, and we definitely would need to fix that first before dealing with stuff like the sample cards (which might actually be fixed by fixing the underlying problem anyway). Can you confirm that?

@bernardsilenou This is not about the person appearing in the person directory, it's about Adina copying & pasting the URL of the person/case in order to get access to it.

**Comment by adinaflorea9** [comment]
@MateStrysewske  That is correct, the person that is editable does not have any entity in the jurisdiction of the user, nor has any of the entities been created by the logged in user.

This ticket covers the problem with the person page being editable - https://github.com/hzi-braunschweig/SORMAS-Project/issues/11245

**Comment by MateStrysewske** [comment]
@adinaflorea9 Thanks for the clarification! I suggest we put this issue on hold and see what happens when #11245 has been done.
